src/components/header/Header.scss

```diff
@@ -1,15 +1,48 @@
 @import "../bootstrap/core";
 
 .header {
+  position: relative;
+  top: 0;
+  left: 0;
```

Comment:
this has to have a height of `7vh` otherwise there is a tiny border a the bottom of page.

Reply:
Yes, that is true. I did not touch it today. `7vh` makes the toolbar to shrink on small-height screen as well as any other viewport-related value because screens are different.

App bar should be fixed size for all screens. Layout issues should be resolved in other way. I can do it but not as a part of this PR. 

Could you please, list files or components I have to take a look at to fix the issue?

Reply:
It's [MainArea](https://github.com/oacore/reader/blob/master/src/components/main-area/MainArea.scss#L6) which wraps all other components.

Reply:
Imho this should solve it. 

```
.main-area {
	position:
    top: 0;
    left: 0;
    bottom: 0;
    right: 0;
}
```